QuizApp/App/Screens/Quiz session/QuizSessionViewController.swift

```diff
@@ -10,14 +10,15 @@ final class QuizSessionViewController: UIViewController {
 
     }
 
-    private var questionNumber = CurrentValueSubject<Int, Never>(1)
+    @Published private var isCorrect: Bool?
+    @Published private var questionNumber = 1
```

Comment:
You've introduced a lot of room for bugs with these publishers. Please check [my branch](https://github.com/IvanKulundzic/QuizApp/compare/feature/quiz-session-api-pt2...feature/quiz-session-api-pt2-marin?expand=1) where I made some changes which we can discuss tomorrow.